.github/workflows/test.yml

```diff
@@ -2,6 +2,8 @@ name: Pull request
 on:
   pull_request:
     types: [opened, synchronize]
+    paths-ignore:
+      - '**.md'
```

Comment:
We *do* want to run the linter when Markdown files change, so it might be easiest to split the linting step into a separate workflow without this option.

Reply:
Also, don't think it's possible to set the [branch protection rules](https://github.com/linz/geostore/settings/branch_protection_rules/17596251) to deal with the new situation. You probably need to add a [`finalize` step](https://github.com/linz/bde-processor-deployment/blob/81e5e5d3a25b2430c1e5b13569a4da82b1f47b1d/.github/workflows/test.yml#L276-L287) and use [`allowed-skips`](https://github.com/re-actors/alls-green) for the tests.

Reply:
Forgotten about `.md` linting. Probably jumped the gun a bit wondering why we trigger long running (>30 min) tests deploying infrastructure to AWS for simple `.md` change. Linting is needed yep. Will have a rethink.